package/Dockerfile

```diff
@@ -5,11 +5,11 @@ USER 1000
 
 FROM base AS copy_dapper
 ONBUILD ARG ARCH
-ONBUILD COPY bin/fleetcontroller-linux-${ARCH} /usr/bin/fleetcontroller
+ONBUILD COPY bin/fleetcontroller-linux-"${ARCH}" /usr/bin/fleetcontroller
```

Comment:
This is not a shell script. Quotes are a bit uncommon in Dockerfiles and I always assumed they are not necessary. Well, unless you EXEC something or RUN a shell script. 
See docs: https://docs.docker.com/engine/reference/builder/#environment-replacement

Here is a blog article telling us not to quote, if you want to know more: https://dev.to/tvanantwerp/don-t-quote-environment-variables-in-docker-268h